vspace/src/main/java/edu/asu/diging/vspace/referenceExpose/ICitationStyle.java

```diff
@@ -0,0 +1,7 @@
+package edu.asu.diging.vspace.referenceExpose;
+
+public interface ICitationStyle {
+    
+    String citationStyleName = null;
```

Comment:
what is this for?

Reply:
?

Reply:
it seems to be implemented by someone else and wasn't useful for the references thus I removed that